analyzer/src/test/kotlin/managers/GoModTest.kt

```diff
@@ -36,10 +36,10 @@ class GoModTest : WordSpec({
             id.toVcsInfo().type shouldBe VcsType.GIT
         }
 
-        "return null as resolved revision" {
+        "return an unresolved revision" {
```

Comment:
Just a note: Personally I'm fine with the wording. But given the past discussions about the meaning of "resolved", we might not want to use "unresolved" here. Maybe just "return the revision"?